### PR TITLE
Support prometheus as a metrics interface

### PR DESCRIPTION
**Pull request by MaratGumarov**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

### CODE REVIEW

**Comment by Haarolean** [comment]
@germanosin PTAL

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/KafkaCluster.java [hunk 1]
```diff
@@ -16,6 +16,7 @@ public class KafkaCluster {
   private final String name;
   private final String version;
   private final Integer jmxPort;
```
Comment (germanosin) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/KafkaCluster.java:18]: I would suggest to rename:
1. Metrics
1.1 port
1.2 type: JMX/PrometheusEndpoint
1.3 basicAuth
1.3.1 username
1.3.2 password
1.4 ssl 
1.4.1 enabled true
1.4.2 truststore
1.4.3 keystore

**Review by germanosin — CHANGES_REQUESTED** [review]

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/JmxExporterMetricsParser.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import java.util.Collections;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class JmxExporterMetricsParser {
```
Comment (germanosin) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/JmxExporterMetricsParser.java:15]: PrometheusEndpointMetricsParser?

**Review by Haarolean — APPROVED** [review]

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/metrics/WellKnownMetrics.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.kafka.common.Node;
+
+class WellKnownMetrics {
+
+  final Map<String, BigDecimal> bytesInFifteenMinuteRate = new HashMap<>();
+  final Map<String, BigDecimal> bytesOutFifteenMinuteRate = new HashMap<>();
+
+  void feed(KafkaCluster cluster, Node node, RawMetric rawMetric) {
```
Comment (Haarolean) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/metrics/WellKnownMetrics.java:18]: feed -> populate ?

**Review by Haarolean — APPROVED** [review]

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/metrics/WellKnownMetrics.java (hunk 3 above)
Comment (iliax) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/metrics/WellKnownMetrics.java]: done

**Pull request merged**